mne/io/base.py

```diff
@@ -1483,6 +1483,7 @@ def export(self, fname, fmt='auto', physical_range='auto',
 
         Supported formats: EEGLAB (set, uses :mod:`eeglabio`)
         %(export_warning)s :meth:`save` instead.
+        %(export_warning_proj)s :meth:`apply_proj`.
```

Comment:
This level of detail seems better suited to the Notes section. Putting too many warnings or notes in the beginning makes it hard for people to get to the params section. I would modify `export_warning` to say `See Notes for details` then in Notes you can have the text about projections being one thing that will not be saved. This part I think you can do without having it in a `.. warning::` block anymore, too.